x/ecocredit/server/marketplace/update_sell_orders.go

```diff
@@ -45,10 +45,13 @@ func (k Keeper) UpdateSellOrders(ctx context.Context, req *marketplace.MsgUpdate
 func (k Keeper) applySellOrderUpdates(ctx context.Context, order *api.SellOrder, update *marketplace.MsgUpdateSellOrders_Update) error {
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
 	var creditType *ecoApi.CreditType
-	event := marketplace.EventUpdateSellOrder{}
+	event := marketplace.EventUpdateSellOrder{
+		Seller:            sdk.AccAddress(order.Seller).String(),
+		SellOrderId:       order.Id,
+		DisableAutoRetire: update.DisableAutoRetire,
+	}
 
 	order.DisableAutoRetire = update.DisableAutoRetire
-	event.DisableAutoRetire = update.DisableAutoRetire
```

Comment:
I think we should remove this from the event but in the current definition of the event, this is one of the updated fields, so this should not have been removed given the current event definition.

Reply:
yeah i had removed it originally in this PR, but during a standup it was said we should keep fields that are updated in events so brought (most) of the fields back. 

ill just axe it.